### ISSUE TITLE
Acces https

### ISSUE THREAD

**Issue by hidayatasep43** [android-async-http/android-async-http#1259]
Hello ..... 
i use android-async-http to access https but i get some error hostname in certificate didn't match <apiem.xyz.id> != <www.abc.com> OR <www.abc.com> 

This is my code 

```java
       AsyncHttpClient client;
       client = new AsyncHttpClient();
        client.setTimeout(10 * 1000);
        client.setMaxRetriesAndTimeout(3, 10 * 1000);
        if (getMethod() == Method.POST)
            client.post(url, params, getAsyncResponseHandler());
        else
            client.get(url, params, getAsyncResponseHandler());
            Timber.e(client.toString());
```

Please help me :)

**Comment by timstableford** [comment]
I have this issue too. It seems to be because of an upstream error in the Apache HTTP client used by this library, so no SNI support. The project also appears to be abandoned.

**Comment by hidayatasep43** [comment]
thanks @timstableford 👍

**Comment by smarek** [comment]
SNI request will be solved in #991, closing this as duplicate
